src/view-backend-private.cpp

```diff
@@ -92,10 +94,10 @@ void ViewBackend::exportEGLStreamProducer(struct wl_resource* bufferResource)
 
 void ViewBackend::dispatchFrameCallbacks()
 {
-    if (G_LIKELY(m_bridgeId))
-        WS::Instance::singleton().dispatchFrameCallbacks(m_bridgeId);
-
-    wpe_view_backend_dispatch_frame_displayed(m_backend);
+    if (G_LIKELY(!m_bridgeIds.empty())) {
```

Comment:
Is it really likely, though? Yeah, there's *probably* been a process swap, but if not it will tank performance, right? I don't like using G_LIKELY() unless I'm confident the case is extremely rare.

Reply:
The most common case is that there is at least one `bridgeId` that associates the surface with a `ViewBackend`; this code is hit for every rendered frame and that is my reasoning for using `G_LIKELY()` here.